lib/main.dart

```diff
@@ -12,6 +13,7 @@ void main() => runApp(const MyApp());
 class MyApp extends StatelessWidget {
   const MyApp({Key? key}) : super(key: key);
 
+// Carlo
```

Comment:
Well structured. Includes important settings like the title, theme, and routes, which makes it easy to manage the app's global configuration.